lib/pbench/test/unit/server/conftest.py

```diff
@@ -129,7 +128,18 @@ def server_config(on_disk_server_config, monkeypatch) -> PbenchServerConfig:
 
 
 @pytest.fixture()
-def client(server_config, fake_email_validator):
+def fake_well_known_auth_endpoints(monkeypatch):
```

Comment:
Is there a need for this to be a separate fixture? Since it's currently only being used by `client`, do we really need it to be separately available or should it be integrated into `client`??